docs/layers.md

```diff
@@ -1,116 +1,510 @@
-# Layers in the vector tile service
+# Layers in Mapzen's vector tiles
 
-Data is organized into several layers comprising the elements typically used for base map rendering. This is a simplified view of OSM data for easier consumption, with common tags often condensed into a single field as noted below.
+![image](images/mapzen-vector-tile-docs-all-layers.png)
 
-## Water
-Polygons representing oceans, riverbanks and lakes. Derived from a combination of the `waterway`, `natural`, and `landuse` tags. Includes coastline-derived water polygons from [openstreetmapdata.com](http://openstreetmapdata.com) at higher zoom levels, and [Natural Earth](http://naturalearthdata.com) polygons at lower zoom levels.
+The [Mapzen vector tile service](https://mapzen.com/projects/vector-tiles) provides worldwide basemap coverage sourced from [OpenStreetMap](www.openstreetmap.org) and other open data projects, updated daily as a free & shared service.
 
-Layer name: `water`
+Data is organized into several layers, grouped into an `all` layer comprising the elements typically used for base map rendering. This is a simplified view of OpenStreetMap data for easier consumption, with common tags often condensed into a single `kind` field as noted below.
```

Comment:
I think the "grouped into an all layer" might be confusing to people. Perhaps it could be expanded to something like:

> Data is organized into several thematic layers, each of which is named, for example; `buildings`, `pois`, `water`. A selection of these layers are typically used for base map rendering, and are provided under the short-hand name `all`.